tools/download-deps.sh

```diff
@@ -119,3 +119,6 @@ done
 
 echo "Installing golangci-lint"
 curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.44.0
+
+echo "Installing Tilt for local development"
+curl -fsSL https://raw.githubusercontent.com/tilt-dev/tilt/v0.23.8/scripts/install.sh | bash
```

Comment:
Please don't `install.sh | bash`, or at least check for `tilt` first.

Reply:
I had tilt in my path: it was not found. The script tried to reinstall and prompted for sudo password (whole other issue on it's own).

Reply:
@morancj Fixed, PTAL

Reply:
shouldn't the URL be `curl -fsSL https://raw.githubusercontent.com/tilt-dev/tilt/master/scripts/install.sh | bash` ?
https://docs.tilt.dev/install.html 🤔

Reply:
I didn't want to install any arbitrary version, as `master` is mutable. The idea was to pin the version for everyone.